apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -17,6 +17,10 @@ import {ReviewStates} from '@cdo/apps/templates/feedback/types';
 import BubbleBadge, {BadgeType} from './BubbleBadge';
 import i18n from '@cdo/locale';
 
+// Duplicated in CommitsAndReviewTab because of import issue,
+// update there as well if changed.
+const URL_PARAMS_TO_EXCLUDE = ['showReviewTab'];
```

Comment:
I have no idea why I needed to do this, but if I tried an import of the constant from `CommitsAndReviewTab`, I successfully imported it (ie, could log the value of the string), use it to create the URLs for the bubbles, but then `JavalabConsole` would fail to render with errors like:

```
javalab.js:36650 Uncaught TypeError: _locale2.default.newFile is not a function
    at RadiumEnhancer.render (javalab.js:36650:34)
    at RadiumEnhancer.render (vendors.js:58286:139)
    at finishClassComponent (vendors.js:82116:31)
    at updateClassComponent (vendors.js:82066:24)
    at beginWork (vendors.js:83576:16)
    at HTMLUnknownElement.callCallback (vendors.js:65144:14)
    at Object.invokeGuardedCallbackDev (vendors.js:65193:16)
    at invokeGuardedCallback (vendors.js:65248:31)
    at beginWork$1 (vendors.js:88159:7)
    at performUnitOfWork (vendors.js:87110:12)
render @ javalab.js:36650
render @ vendors.js:58286
finishClassComponent @ vendors.js:82116
updateClassComponent @ vendors.js:82066
beginWork @ vendors.js:83576
callCallback @ vendors.js:65144
invokeGuardedCallbackDev @ vendors.js:65193
invokeGuardedCallback @ vendors.js:65248
beginWork$1 @ vendors.js:88159
performUnitOfWork @ vendors.js:87110
workLoopSync @ vendors.js:87086
performSyncWorkOnRoot @ vendors.js:86712
scheduleUpdateOnFiber @ vendors.js:86144
updateContainer @ vendors.js:89329
(anonymous) @ vendors.js:89714
unbatchedUpdates @ vendors.js:86859
legacyRenderSubtreeIntoContainer @ vendors.js:89713
render @ vendors.js:89796
push../src/javalab/Javalab.js.Javalab.init @ javalab.js:35063
onReady @ common.js:77461
setTimeout (async)
push../src/dom.js.exports.addReadyListener @ code-studio-common.js:149950
_default @ common.js:77487
loadJavalab @ javalab.js:41022
Promise.then (async)
./src/sites/studio/pages/levels-javalab-main.js @ javalab.js:41045
__webpack_require__ @ webpack-runtime.js:85
14 @ javalab.js:41442
__webpack_require__ @ webpack-runtime.js:85
checkDeferredModules @ webpack-runtime.js:47
webpackJsonpCallback @ webpack-runtime.js:34
(anonymous) @ javalab.js:1
javalab.js:35684 Uncaught TypeError: _locale.default.console is not a function
    at JavalabConsole.render (javalab.js:35684:27)
    at finishClassComponent (vendors.js:82116:31)
    at updateClassComponent (vendors.js:82066:24)
    at beginWork (vendors.js:83576:16)
    at HTMLUnknownElement.callCallback (vendors.js:65144:14)
    at Object.invokeGuardedCallbackDev (vendors.js:65193:16)
    at invokeGuardedCallback (vendors.js:65248:31)
    at beginWork$1 (vendors.js:88159:7)
    at performUnitOfWork (vendors.js:87110:12)
    at workLoopSync (vendors.js:87086:22)
```